lib/cli.js

```diff
@@ -220,35 +220,32 @@ const meowOptions = {
         When glob pattern matches no files, the process will exit without throwing an error.
 	`,
 	flags: {
-		'allow-empty-input': {
+		allowEmptyInput: {
```

Comment:
Why convert these keys to kebab case?

Reply:
Assume the following example:

```js
// cli-example.js
const meow = require('meow');

const cli = meow({
  flags: {
    'max-warnings': {
      type: 'number'
    },
  },
});

console.log(cli.flags['max-warnings']);
console.log(cli.flags.maxWarnings);
```

When running the script, we can get this result:

```shell-session
$ node cli-example.js --max-warnings=1
undefined
1
```

But the `meow.Result` type has an incorrect `max-warnings` property instead of `maxWarnngs`.

<img width="944" alt="image" src="https://user-images.githubusercontent.com/473530/72116270-21e0b000-338d-11ea-954b-71b0742df203.png">

When converting to camel-case:

```diff
  flags: {
-   'max-warnings': {
+   maxWarnings: {
      type: 'number'
```

We can get the correct type:

<img width="901" alt="image" src="https://user-images.githubusercontent.com/473530/72116578-ff02cb80-338d-11ea-8564-34c810498b5c.png">